### PR TITLE
Set cri-o minor version due to issue with latest release

### PR DESCRIPTION
**Pull request by brianmcarey**
The latest minor releases of cri-o fail to start successfully in
kubevirtci. This change sets the cri-o version to the previous minor
release.

kubevirtci looks to be hitting the following cri-o issue - https://github.com/cri-o/cri-o/issues/5841

/cc @dhiller @xpivarc 

Signed-off-by: Brian Carey <bcarey@redhat.com>

### CODE REVIEW

**Comment by xpivarc** [comment]
@brianmcarey Seems the crio issue is gone but we have some consistent failures.
@EdDev Would you be able to have a look?

**Comment by brianmcarey** [comment]
@xpivarc these consistent failures should be resolved by https://github.com/kubevirt/kubevirtci/pull/795 - that PR is blocked by the cri-o issue so once the changes in this PR are approved I will override the four failed check-provision-k8s-* jobs.

**Comment by xpivarc** [comment]
> @xpivarc these consistent failures should be resolved by #795 - that PR is blocked by the cri-o issue so once the changes in this PR are approved I will override the four failed check-provision-k8s-* jobs.

Thanks for pointing it out! We can probably override the #795 as there are no provision changes afaik that would unblock this.

**Comment by oshoval** [comment]
Thanks, lets retest please now that 795 is merged

**Comment by brianmcarey** [comment]
> Thanks, lets retest please now that 795 is merged don't we need it also on k8s-1.24 ?

the [publish-kubevirtci](https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/logs/publish-kubevirtci/1523942430078930944) job failed due to the cri-o issue so the required image didn't get pushed.

**Comment by oshoval** [comment]
> > Thanks, lets retest please now that 795 is merged don't we need it also on k8s-1.24 ?
> 
> the [publish-kubevirtci](https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/logs/publish-kubevirtci/1523942430078930944) job failed due to the cri-o issue so the required image didn't get pushed.

Thanks, just saw it as well

**Comment by brianmcarey** [comment]
The required images was pushed manually

/retest-required

**Comment by brianmcarey** [comment]
All check-provision-k8s-* jobs are passing - @dhiller @xpivarc - can you please take a look?

**Review by xpivarc — COMMENTED** [review]
I am just wondering if we don't want to adjust mirror-crio.sh too... and rather use our mirror than maybe not so stable public repository(or maybe we will use extensively the public API ).

**Comment by brianmcarey** [comment]
> I am just wondering if we don't want to adjust mirror-crio.sh too... and rather use our mirror than maybe not so stable public repository(or maybe we will use extensively the public API ).

Thanks @xpivarc  - I will take a look into this.

**Comment by brianmcarey** [comment]
Failures expected as the kubevirtci mirror for the previous minor release of cri-o is not in place yet - https://github.com/kubevirt/project-infra/pull/2086

**Review by dhiller — APPROVED** [review]
/approve

**Comment by brianmcarey** [comment]
it looks like the previous minor releases buids have been bumped and now we are are hitting the crio issue on the previous release too.

Current failing run:
```shell
Installing:
 cri-o                          x86_64  1.22.2-2.17.el8                         devel_kubic_libcontainers_stable_cri-o_1.22_1.22.2   23 M
```

the successful run on this PR:
```shell
Installing:
 cri-o                          x86_64  1.22.2-2.5.el8                          devel_kubic_libcontainers_stable_cri-o_1.22_1.22.2   25 M
```

/hold

**Comment by brianmcarey** [comment]
Issue is resolved with latest release of cri-o

**Pull request closed without merging**
